SUPPORT.md

```diff
@@ -0,0 +1,77 @@
+<span>
+  Join our Discord server to chat with our team directly and become a part of the Passage community.
+  <a href="https://discord.com/invite/445QpyEDXh">Join Discord →</a>
+</span>
+
+![Static Badge](https://img.shields.io/badge/support%40passage.id-%233B66BC?logo=1Password&logoColor=white) <br />
+<span>
+  Send an email to support@passage.id and we'll respond to your inquiry as soon as possible.
+  <a href="mailto:support@passage.id">Email Passage →</a>
+</span>
+
+![Static Badge](https://img.shields.io/badge/GitHub-%23181717?logo=github&logoColor=white) <br />
+<span>
+  Open an issue on any of our GitHub repositories and we'll repond to your inquiry as soon as possible.
+</span>
+
+## Browse our Documentation
+Explore our products and determine how you want to start using Passage by 1Password at [docs.passage.id](https://docs.passage.id). Each product has a rich documentation presence with code snippets and guides to help you find all of the resources you need to easily integrate Passage.
+
+<p>
+    <img src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png"> Passkey <b>Flex</b><br />
+    <b>Add passkeys to an existing auth solution:</b>
+    Upgrade your existing authentication flow to support passkeys. With Passkey Flex, you can choose how you incorporate passkeys: as an alternative to passwords at login and registration, as a multi-factor authentication method, or allow users to add a passkey to an existing account.
+    <a href="https://docs.passage.id/flex">Passkey Flex Docs →</a>
+</p>
+
+<p>
+    <img src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png"> Passkey <b>Complete</b><br />
+    <b>Use Passage as a standalone auth solution:</b>
+    Replace your existing authentication flow or build from scratch with a robust solution for passwordless authentication and customer identity management. In addition to passkeys, you can add other authentication methods including social sign-in, Magic Links, and one-time codes.
+    <a href="https://docs.passage.id/complete">Passkey Complete Docs →</a>
+</p>
+
+<p>
+    <img src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png"> Passkey <b>Ready</b> <br />
+    <b>Determine if your users are ready for passkeys:</b>
+    Add a few lines of code to your app or website and Passkey Ready will generate data to determine if your end users are ready for passkeys.
+    <a href="https://docs.passage.id/passkey-ready">Passkey Ready Docs →</a>
+</p>
+
+## Request a Demo
+We're always eager to chat! Whether you're looking for a demo or just have some questions about our products, please [reach out](https://passage.1password.com/contact) and we’ll be in touch shortly.
+
+## Additional Resources
+
+* [Blog](https://passage.1password.com/blog)
+* [Console](https://console.passage.id)
+* [Status](https://status.passage.id/)
+* [Security]()
```

Comment:
Missing link

Reply:
@agebhardt where should this link?

Reply:
This should link to the new `SECURITY.md` file that's being added to this repo.